submitit/slurm/test_slurm.py

```diff
@@ -343,3 +343,30 @@ def test_slurm_invalid_parse() -> None:
 def test_slurm_missing_node_list() -> None:
     with with_slurm_job_nodelist("") as env:
         assert [env.hostname] == env.hostnames
+
+
+def test_slurm_weird_dir(tmp_path: Path, weird_dir: str) -> None:
+    if "\n" in weird_dir:
+        pytest.skip("test doesn't support newline in 'weird_dir'")
```

Comment:
is it supported by slurm? if not then raise there maybe?

Reply:
I believe that newline in filename is supported by sbatch but not by my crappy sbatch parsing test code. Can I change the message to make this more clear ?